packages/payload/src/admin/components/elements/ListDrawer/DrawerContent.tsx

```diff
@@ -215,6 +215,15 @@ export const ListDrawerContent: React.FC<ListDrawerProps> = ({
     return null
   }
 
+  const listComponent = selectedCollectionConfig?.admin?.components?.views?.List
+
+  const ListToRender =
```

Comment:
Please no nested ternaries.
Here is a chat-gpt written alternative to L218-L226:
```
const listComponent = selectedCollectionConfig?.admin?.components?.views?.List;
let ListToRender = null;

if (listComponent && typeof listComponent === 'function') {
  ListToRender = listComponent;
} else if (
  typeof listComponent === 'object' &&
  typeof listComponent.Component === 'function'
) {
  ListToRender = listComponent.Component;
}
```